.github/dependabot.yaml

```diff
@@ -16,3 +16,8 @@ updates:
       interval: "daily"
     labels:
       - "dependencies"
+    # Not yet supported. See <https://github.com/dependabot/dependabot-core/issues/4009>.
+    # versioning-strategy: "increase-if-necessary"
+    ignore:
+      - dependency-name: "serde"
+        update-types: ["version-update:semver-minor", "version-update:semver-patch"]
```

Comment:
Could you also add ignores for patch version of all crates e.g. currently dependabot is trying to bump anyhow and parking lot as well, which is unneeded.